apps/tx-builder/src/App.tsx

```diff
@@ -0,0 +1,60 @@
+import { Routes, Route, useNavigate } from 'react-router-dom';
+import Header from './components/Header';
+
+import useTransactions from './hooks/useTransactions';
+import Dashboard from './pages/Dashboard';
+import ReviewAndConfirm from './pages/ReviewAndConfirm';
+import { HOME_PATH, REVIEW_AND_CONFIRM_PATH } from './routes/routes';
+
+const App = () => {
```

Comment:
I created this new `App` component with all the Tx-Builder routes and to handle the state of transactions
:3